app/client/src/features/organizations/OrgList.tsx

```diff
@@ -85,12 +86,10 @@ export const OrgList = ({ queryRef }: OrgListProps) => {
     const data = usePreloadedQuery(ORG_LIST_QUERY, queryRef);
     const classes = useStyles();
     const router = useRouter();
+    const initialState = {id: '', name: ''};
```

Comment:
I'd declare this outside of `OrgList`.  With the way react works, it will re-declare this variable on every re-render.

```js
const intiailState = { ... }
export const OrgList = {
   // ... 
  const [selectedOrg, setSelectedOrg] = React.useState(initialState);
}
```